Formula/convox.rb

```diff
@@ -0,0 +1,15 @@
+class Convox < Formula
+  desc "The convox AWS PaaS CLI tool"
+  homepage "https://convox.com/"
+  url "https://bin.equinox.io/a/9RUvivxx6Lm/convox-20160606232025-darwin-amd64.tar.gz"
```

Comment:
I tried doing a cask first, was asked to take it here because it's a CLI only tool. It's a go binary, single file inside the binary - not sure if that's similar enough to a jar or not. 

Are there any good go formulas out there that I can model this on? Not too familiar with how to build it from source.